pkg/dockerregistry/server/auth_test.go

```diff
@@ -279,6 +277,11 @@ func TestAccessController(t *testing.T) {
 		ctx := context.WithValue(context.Background(), "http.request", req)
 
 		server, actions := simulateOpenShiftMaster(test.openshiftResponses)
+		accessController, err := newAccessController(options)
+		if err != nil {
+			t.Fatal(err)
+		}
+		accessController.(*AccessController).config = kclient.Config{Host: server.URL, Insecure: true}
```

Comment:
is there a better way to inject this earlier, the way we'd actually configure the client? Would moving up the `DefaultRegistryClient` assignment below let us drop this?